src/types/TodoContext.ts

```diff
@@ -0,0 +1,6 @@
+import { Todo } from './Todo';
+
+export type TodoContextType = {
```

Comment:
```suggestion
export interface TodoContextType  {
```
use interface prefer